docs/qcforward/bw_vs_gridprops.inc

```diff
@@ -0,0 +1,183 @@
+.. _qcforward-bwvsprops:
+
+Compare blocked wells with corresponding grid properties
+--------------------------------------------------------
+
+This method checks how the blocked well cell properties match with corresponding grid
+properties in the 3D grid. If worse than a given set of limits, either a warning is
+given or a full stop of the workflow is forced.
+
+The input to this method is a python dictionary with some defined keys. Note that
+the order of keys does not matter. It is advised to study the examples further below.
+
+
+Common fields (same input inside or outside RMS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+verbosity
+  Level of output while running None, "info" or "debug", default is None (optional).
+
+compare:
+  A dictionary on the form ``{"Facies": "FACIES", "PHIT": "POROSITY"}`` where the first
+  elements are the property name in blocked wells (here ``"Facies, "PHIT"``) and the second
+  elements (after the colon) are the corresponding name in the grid properties (here
+  ``"FACIES", "POROSITY"``). Note that names can of course be equal.
+
+
+actions
+  This is a list of dictionaries that shows what actions which shall be performed per
+  well (any) or for the average (all), for example ``{"warn": "all < 50", "stop": "all <
+  30"}`` which means that match < than 50% will trigger a warning, while a match < 30%
+  will trigger a stop in work flow. Note that ``any`` and ``all`` shall not be mixed in
+  the same record (required). Also both lines are (currently) required.
+
+report
+  Result will be written in a CSV file (which e.g. can be used in plotting) on disk.
+  (optional).
+
+dump_yaml
+  If present, should be a file name where the current data structure is dumped to YAML
+  format. Later this YAML file can be edited and applied for a single line input
+  (optional).
+
+nametag
+  A string to identify the data set. Recommended.
+
+show_data
+  A string (or bool) if the full dataset shall be shown to screen as a dataframe
+  (table). This can be useful when investigating what wells that triggers e.g. a stop.
+  Possible values are False (default, means no output), "warn" to wells with WARN or
+  STOP criteria, "fail" to see wells that fails only (due to STOP) and True or "all" to
+  show everything. Any other words (like "foo") will be interpreted as "all").
```

Comment:
Would it be better to have this as an optional argument with default value None? That would reduce the list of possible values to `warn`/`fail`/`all` and make the documentation a bit simpler.